### PR TITLE
Component name fixing, createproduct-card and modifide some file,style

### PR DESCRIPTION
**Pull request by rahmanshifur**

### CODE REVIEW

**Review by SohagHossain9983 — APPROVED** [review]
very good

File: components/breadcrumb/index.jsx [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
-const Breadcrumb = () => {
-    return(
-        <div className="breadcrumb">
+const Breadcrumb = (props) => {
+    return (
+        <div id="breadcrumb">
+            <div className="container">
+                <div className="shop-route">
+                    {/* <Link> HOME </Link> */}
+                    <a href="#">{props.mainLink}</a>
```
Comment (naimuddin374) [inline components/breadcrumb/index.jsx]: add linktag

File: components/filter-bar/filter-checkbox.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    return (
+        <div className="checkbox">
+            <h4>{props.heading}</h4>
+            <div>
+                <Input
+                    className="shadow-none"
+                    type="checkbox"
+                    id={props.CheckId}
+                />
+                <Label for={props.CheckId} role="button" className="ms-2 mb-0">{props.name}</Label>
```
Comment (naimuddin374) [inline components/filter-bar/filter-checkbox.jsx:13]: replace for to htmlFor

File: components/pagination/pagination-number.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+
+const NumberPagination = () => {
```
Comment (naimuddin374) [inline components/pagination/pagination-number.jsx]: fix component

File: components/pagination/prev-button.jsx [hunk 4]
```diff
@@ -0,0 +1,8 @@
+
+
+const PrevButton = () => {
+    return (
+        <div className="priv"><a className="px-2" href="#">Priv</a></div>
```
Comment (naimuddin374) [inline components/pagination/prev-button.jsx]: Use link instead of a tag

File: styles/components/pagination/index.scss [hunk 5]
```diff
@@ -4,17 +4,29 @@
     .pagination-body {
```
Comment (naimuddin374) [inline styles/components/pagination/index.scss:4]: change file name

File: styles/components/shop-bar/index.scss [hunk 6]
```diff
@@ -1,33 +1,10 @@
...
-                justify-content: space-between;
-
-            }
-
-        }
-
-    }
-}
\ No newline at end of file
+ #shop-bar {
```
Comment (naimuddin374) [inline styles/components/shop-bar/index.scss:1]: change filename

**Review by naimuddin374 — CHANGES_REQUESTED** [review]
change

**Review by naimuddin374 — APPROVED** [review]

**Pull request merged**